Source: mscorlib
Message: The operation completed successfully.

HelpLink: 
Stack Trace =>
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.__Error.WinIOError()
   at System.Console.set_Title(String value)
   at FireRTCBot.Program.<RunAsync>d__17.MoveNext() in C:\Users\Craftplacer\Google Drive\Coding Projects (GitHub)\FireRTC-Spammer\FireRTCSpammer\Program.cs:line 195
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at FireRTCBot.Program.Main(String[] args) in C:\Users\Craftplacer\Google Drive\Coding Projects (GitHub)\FireRTC-Spammer\FireRTCSpammer\Program.cs:line 47
<=
